packages/nimble-blazor/NimbleBlazor.Acceptance.Tests/PlaywrightFixture.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.Playwright;
+using Xunit;
+using PlaywrightProgram = Microsoft.Playwright.Program;
+
+namespace NimbleBlazor.Tests.Acceptance;
+
+/// <summary>
+/// Fixture to handle Playwright initialization for acceptance tests.
+/// </summary>
+public class PlaywrightFixture : IAsyncLifetime
+{
+    private IBrowser? _browser;
+    private IPlaywright? _playwright;
+    public IBrowserContext? BrowserContext { get; private set; }
+
+    public async Task InitializeAsync()
+    {
+        await InstallChromiumAsync();
+        _playwright = await Playwright.CreateAsync();
+        _browser = await _playwright.Chromium.LaunchAsync(
+            new BrowserTypeLaunchOptions()
+            {
+#if DEBUG
+                Headless = false,
+                SlowMo = 1000
+#endif
+            });
+        BrowserContext = await _browser.NewContextAsync(new BrowserNewContextOptions { IgnoreHTTPSErrors = true });
+#if DEBUG
+        BrowserContext.SetDefaultTimeout(30000);
+#endif
+    }
+
+    public async Task DisposeAsync()
+    {
+        if (BrowserContext != null)
+        {
+            await BrowserContext.DisposeAsync();
+        }
+        if (_browser != null)
+        {
+            await _browser.DisposeAsync();
+        }
+        _playwright?.Dispose();
+    }
+
+    private Task InstallChromiumAsync()
+    {
+        return Task.FromResult(PlaywrightProgram.Main(new string[] { "install", "chromium" }));
```

Comment:
Can we instead use the playwright version we get with npm already?

Might need to use a strict version of playwright in the nimble-blazor package.json and create a simple node script to validate the the package.json playwright version aligns with the csproj playwright version.

Reply:
I'll look into that. I originally diverged from Playwright 1.30 -> 1.33 when I was debugging some intermittent test issues, but I think we should be able to use the same version.

Reply:
These tests now use Playwright v1.30 which matches the version nimble-components is using. So both use the same Chromium version (110.0.5481.38).

Keeping this comment thread open as I haven't added the script that you mentioned to validate the versions match yet.

Reply:
I'm in agreement with @rajsite (I think), and to check my understanding, I think what we want is something like a post-build step that essentially runs `playwright install`, correct?

I'll note for Milan, that the pattern of installing the Chromium browser to use while running Playwright tests [isn't something we're inventing](https://www.meziantou.net/automated-ui-tests-an-asp-net-core-application-with-playwright-and-xunit.htm#automating-the-brows), but the reasoning they give for when you would do that is if you were distributing an application that depended on Playwright, which is not the case here.

Reply:
> I think what we want is something like a post-build step that essentially runs playwright install, correct?

I don't think we even need that. My working understanding is that adding "playwright": "some-version" to the package.json already accomplishes the exact same thing and we can avoid the extra work

Reply:
Updated:
- nimble-blazor package.json specifies Playwright v1.30 as a devDependency
- nimble-blazor `postinstall` script generates `Playwright.PackageVersion.props` based on the resolved Playwright npm package version (so it's linking the Playwright NuGet version to the Playwright npm package version)
- NimbleBlazor.Tests.Acceptance.csproj imports that props file and uses it for the Playwright PackageReference
- Removed 'chromium install' steps from PlaywrightFixture